test/integration/kubeadm/util.go

```diff
@@ -1,3 +1,5 @@
+// +build integration
```

Comment:
Generally I think we should discourage and stop using build tags like this. If we use build tags, we probably won't ever check that these tests actually compile.